src/enrichment_facility_tests.cc

```diff
@@ -259,175 +515,10 @@ TEST_F(EnrichmentFacilityTest, ValidReq) {
   EXPECT_TRUE(src_facility->ValidReq(mat));  // valid
 }
 
-// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-TEST_F(EnrichmentFacilityTest, EmptyRequests) {
-  using cyclus::Material;
-  using cyclus::RequestPortfolio;
-
-  src_facility->SetMaxInventorySize(src_facility->InventorySize());
-  std::set<RequestPortfolio<Material>::Ptr> ports =
-      src_facility->GetMatlRequests();
-  ports = src_facility->GetMatlRequests();
-  EXPECT_TRUE(ports.empty());
-}
-
```

Comment:
This tested EnrichmentFacility function GetMatlRequests, which is implicitly tested in MockSim instance